Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -696,6 +698,16 @@ static GridArea insertIntoGrid(Grid& grid, RenderBox& child, const GridArea& are
     return clamped;
 }
 
+static inline bool isMasonryRows(const RenderStyle& style)
+{
+    return style.gridMasonryRows();
+}
+
+static inline bool isMasonryColumns(const RenderStyle& style)
+{
+    return !style.gridMasonryRows() && style.gridMasonryColumns();
```

Comment:
This could also be:

`return !isMasonryRows(stye) && style.gridMasonryColumns();`

Reply:
Changed